website/docs/r/kinesis_log_source.html.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_log_metrics_source"
+description: |-
+  Provides a Sumologic Kinesis Log source. This source is used to integrate with Metrics Stream via Kinesis Firehose from AWS.
+---
+
+# sumologic_kinesis_log_source
+
+Provides a Sumologic Kinesis Log source. This source is used to ingest log via Kinesis Firehose from AWS.
+
+__IMPORTANT:__ The AWS credentials are stored in plain-text in the state. This is a potential security issue.
+
+## Example Usage
+```hcl
+locals {
+
+  resource "sumologic_kinesis_log_source" "kinesis_log_access_key" {
+    name         = "Kinesis Log"
+    description  = "Description for Kinesis Log Source"
+    category     = "prod/kinesis/log"
+    content_type = "KinesisLog"
+    collector_id = "${sumologic_collector.collector.id}"
+    authentication {
+      type       = "S3BucketAuthentication"
+      access_key = "someKey"
+      secret_key = "******"
+    }
+
+    path {
+      type            = "KinesisLogPath"
+      bucket_name     = "testBucket"
+      path_expression = "http-endpoint-failed/*"
+      scan_interval   = 30000
+    }
+  }
+
+  resource "sumologic_kinesis_log_source" "kinesis_log_role_arn" {
+    name         = "Kinesis Log"
+    description  = "Description for Kinesis Log Source"
+    category     = "prod/kinesis/log"
+    content_type = "KinesisLog"
+    collector_id = "${sumologic_collector.collector.id}"
+
+    authentication {
+      type     = "AWSRoleBasedAuthentication"
+      role_arn = "arn:aws:iam::604066827510:role/cw-role-SumoRole-4AOLS73TGKYI"
+    }
+
+    path {
+      type            = "KinesisLogPath"
+      bucket_name     = "testBucket"
+      path_expression = "http-endpoint-failed/*"
+      scan_interval   = 30000
+    }
+  }
+
+  resource "sumologic_collector" "collector" {
+    name        = "my-collector"
+    description = "Just testing this"
+  }
+}
+```
+
+## Argument reference
+
+In addition to the common properties, the following arguments are supported:
+
+ - `content_type` - (Required) The content-type of the collected data. Details can be found in the [Sumologic documentation for hosted sources][1].
+ - `authentication` - (Required) Authentication details for connecting to the S3 bucket.
```

Comment:
This should be optional. And instead of copy pasting directly from existing source page, should explain here what this S3 bucket is intended for and why someone would be interested in setting this up

Reply:
+1 this is marked as Optional in the resource.